docs/pages/blog/mui-x-sep-2024-price-update.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Upcoming changes to MUI X pricing in 2024
+description: Check out the new pricing updates and how to transition to the new model.
+date: 2024-08-01T08:00:00.000Z
+authors: ['josefreitas']
+tags: ['MUI X', 'Product']
+manualCard: true
+---
+
+We're excited to share some important updates to our product offerings and pricing structure. This change reflects the expanded feature set and helps us continue delivering high-quality components to you and your users.
+
+## Changes to the Pro plan
+
+### Current Pricing
+
+Under our current pricing model, if you licensed our product for 50 developers, you only paid for 10 seats. This concept was introduced when we offered only the Data Grid Pro with a much smaller feature set.
+
+### New Pricing
+
+With the launch of several new components, including date and time range pickers, as well as upcoming Pro versions of Charts and TreeView, we're updating our pricing.
+
+Moving forward, each developer requires a seat. So, for example, 50 developers will now need 50 seats.
+
+We're happy to discuss volume discounts to accommodate larger teams.
+
+### Grandfathering Current Customers
+
+We value our loyal customers and want to ensure a smooth transition: current customers will be grandfathered into their existing plans.
+
+This means you can continue using and renewing the legacy license for as long as you wish.
+However, the legacy license covers only Data Grid and Date Pickers.
+
+:::info
+**Access to new components:** The Pro versions of Charts and Tree View, and other future Pro components, will be available exclusively through a license key under the new Pro plan model or a Premium key (Premium users will experience no change).
+::::
+
+### Pricing for Small Teams
+
+The price is essentially the same for customers with 10 developers or fewer, but you still need a new license key to use the Pro versions of Charts or Tree View. If that's the case, we recommend you contact our sales team and get a discount to renew your license earlier.
+
+## Changes to the Premium plan
+
+### Ending the early bird discount
+
+After two years since the introduction of our Premium plan, we've added numerous features and are on the verge of releasing the Pivot mode for the Data Grid. We consider the product mature and battle tested enough and as a result, the early bird discount for the Premium plan finally reaches its end.
```

Comment:
```suggestion
After two years since the [introduction of our Premium plan](/blog/premium-plan-release/), we've added numerous features and are on the verge of releasing the Pivot mode for the Data Grid. We consider the product mature and battle tested enough and as a result, the early bird discount for the Premium plan finally reaches its end.
```